apps/OboeTester/app/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
             path "CMakeLists.txt"
         }
     }
+    ndkVersion '21.4.7075529'
```

Comment:
Do we need this? We don't have this in other parts of our codebase in Oboe.

Reply:
It builds without this. I will remove it.